lsp-completion.el

```diff
@@ -625,17 +627,13 @@ Returns resolved completion item details."
                                                            ((and done? (not (seq-empty-p items)))
                                                             (lsp-completion--to-internal items))
                                                            ((not done?) :incomplete))
-                                                          :lsp-items nil
                                                           :markers markers
                                                           :prefix prefix)
-                              result (lsp-completion--filter-candidates
-                                      (cond (done?
-                                             (cl-second lsp-completion--cache))
-                                            (lsp-completion-filter-on-incomplete
-                                             (lsp-completion--to-internal items)))
-                                      :lsp-items items
```

Comment:
The reason why we use `lsp-items` here (probably I should name it as `foo-items` at that time) is because of a flickering issue in `company-mode`. Thus, when the candidates are not ready or empty, we faked it the previous result and do no such things like filtering to keep the computation code down 
See https://github.com/emacs-lsp/lsp-mode/pull/2148

Reply:
should be fixed now